deploy/infra/lib/constructs/cache-sync.ts

```diff
@@ -37,16 +38,19 @@ export class CacheSync extends Construct {
 
     props.dynamoTable.grantReadWriteData(this._lambda);
 
-    //add event bridge trigger to lambda
-    this.eventRule = new events.Rule(this, "EventBridgeCronRule", {
-      schedule: events.Schedule.cron({ minute: "0/5" }),
-    });
+    if (props.shouldRunAsCron) {
```

Comment:
For both of these rules in cache sync and healthcheck, the if statement means that the rule will not be created in AWS at all when it is disabled.

The achieves the desired effect of being able to configure it, however the downside is that if you were doing development on the lambda cache sync, you need to do a stack deployment to turn syncing on.

This can take 5 - 6 minutes or so.

The alternative is that you can create the rule in CDK and just set it to disabled.
The benefit of this is that as a developer, I can go into the AWS console and just toggle the rule on or off if and when I need to, and I don't need to wait 5 minutes.

Using the config in deployment.yml I can enable the rule permanently.

```
      //add event bridge trigger to lambda
      schedule: events.Schedule.cron({ minute: "0/5" }),	      this.eventRule = new events.Rule(this, "EventBridgeCronRule", {
    });	        schedule: events.Schedule.cron({ minute: "0/5" }),
enabled: props.shouldRunAsCron,
      });

      // add the Lambda function as a target for the Event Rule
      this.eventRule.addTarget(new targets.LambdaFunction(this._lambda));


    // add the Lambda function as a target for the Event Rule	      // allow the Event Rule to invoke the Lambda function
    this.eventRule.addTarget(new targets.LambdaFunction(this._lambda));	      targets.addLambdaPermission(this.eventRule, this._lambda);
```

Reply:
Cheers Josh, solid feedback